package.json

```diff
@@ -95,7 +95,7 @@
     "@babel/core": "7.12.17",
     "@babel/preset-env": "7.12.17",
     "@babel/runtime": "7.12.18",
-    "@nuxt/types": "2.15.0",
+    "@nuxt/types-edge": "latest",
```

Comment:
```suggestion
    "@nuxt/types-edge": "npm:@nuxt/types",
```

Since modules extend `@nuxt/types` directly, this package is exception

Reply:
I think you meant the opposite?

```
"@nuxt/types": "npm:@nuxt/types-edge",
```

Reply:
Yes. Sorry for typo

Reply:
No problem and thanks. Didn't know about such syntax even.